draft-ietf-sidrops-aspa-verification.xml

```diff
@@ -221,70 +216,69 @@
+        If multiple ASPAs exist for a customer AS, then the VAP represents the union of the SPAS from those ASPAs.
       </t>
     </section>
 
-    <section title="ASPA Registration Recommendations" anchor="rec1">
-      <t>
-        An ASPA object showing only AS 0 as a provider AS is referred to as an AS0 ASPA.
-        A non-transparent Route Server AS (RS AS) is one that includes its AS number in the AS_PATH.
-        Registering as AS0 ASPA is a statement by the registering AS that it has no transit providers, and it is also not an RS-client at a non-transparent RS AS.
-        If that statement is true, then the AS MUST register an AS 0 ASPA.
-      </t>
-      <t>
-        Normally, the Provider ASes of a CAS would be congruent for the address family combinations {AFI 1 (IPv4), SAFI 1} and {AFI 2 (IPv6), SAFI 1}.
-        Exceptions to this are expected to be rare.
-        In any case, the CAS MUST list the union of all Provider ASes applicable to the address family combinations stated above in the SPAS and MUST also include any non-transparent RS AS(es) at which it is an RS-client.
-        In the procedures for the AS path verification described in this document (<xref target="pair-validation"/>, <xref target="verif"/>), the SPAS is always considered to be uniformly applicable to {AFI 1 (IPv4), SAFI 1} and {AFI 2 (IPv6), SAFI 1}.
-      </t>
-      <t>
-        A compliant AS, including a Route Server AS (RS AS), MUST have an ASPA.
-        An RS AS SHOULD register an AS 0 ASPA.
-        An AS SHOULD NOT have more than one ASPA.
-        Such a practice helps prevent race conditions during ASPA updates that might affect prefix propagation.
-        The software that provides hosting for ASPA records SHOULD support enforcement of this practice.
-        During a transition process between different certificate authority (CA) registries, the ASPA records SHOULD be kept identical in all relevant registries.
-      </t>
-      <t>
-        As mentioned in <xref target="ASPA"/>, the set of provider ASes contained in the VAP(s) is referred to as the VAP-SPAS of the AS registering the ASPA(s).
-        Normally, a VAP-SPAS is not expected to contain both an AS 0 and other Provider ASes,
-        but an unexpected presence of AS 0 has no influence on the AS path verification procedures (see <xref target="pair-validation"/>, <xref target="verif"/>).
-      </t>
-      <t>
-        Each of the two ASes in a mutual-transit pair MUST register its ASPA including the other AS in its SPAS.
-      </t>
-      <t>
-        The ASes on the boundary of an AS Confederation MUST register ASPAs using the Confederation's global ASN as the CAS.
-      </t>
-    </section>
-    <section title="Provider Authorization Function" anchor="pair-validation">
+      <section title="ASPA Registration Recommendations" anchor="rec1">
```

Comment:
Please check the number of whitespaces that you are using, if you just shift a section, it makes the diff unreadable.